ts/configs/metrics.yaml

```diff
@@ -0,0 +1,34 @@
+dimensions:
+  ModelName: &model_name "model_name"
+  Level: &level "Level"
+
+ts_metrics:
+  counter:
+    - name:
+        unit: ms
+        dimensions: [*model_name, *level]
```

Comment:
please sync up the metric.yaml b/w python and cpp

Reply:
Fixed